tar/bsdtar.c

```diff
@@ -963,7 +963,7 @@ main(int argc, char **argv)
 	network_bwlimit(bsdtar->bwlimit_rate_down, bsdtar->bwlimit_rate_up);
 
 	/* Perform the requested operation. */
-	switch(bsdtar->mode) {
+	switch (bsdtar->mode) {
```

Comment:
Technically this line is libarchive code.  Can you give me a PR for just the tarsnap code, and perhaps send the libarchive style fix upstream?